fspmwheat/caribu_facade.py

```diff
@@ -28,7 +29,7 @@
 SHARED_ELEMENTS_INPUTS_OUTPUTS_INDEXES = ['plant', 'axis', 'metamer', 'organ', 'element']
 
 #: the outputs of Caribu
-CARIBU_OUTPUTS = ['PARa']
+CARIBU_OUTPUTS = ['PARa', 'Erel']
```

Comment:
Erel still used?

Reply:
Yes : light distribution model is run every 4 hours, but every hour we update PARa using hourly PARi and Erel